tests/e2e/e2e_setup_test.go

```diff
@@ -52,9 +51,9 @@ type IntegrationTestSuite struct {
 	valResources   map[string][]*dockertest.Resource
 }
 
-func TestIntegrationTestSuite(t *testing.T) {
-	suite.Run(t, new(IntegrationTestSuite))
-}
+//func TestIntegrationTestSuite(t *testing.T) {
```

Comment:
this is the entry point to test e2e, when i comment out, there is no test to run, if i only comment out ibc test, will also show no test to run. @okwme

Reply:
yeah, i should not comment out, if we still want to run e2e

Reply:
Is this bc of IBC?